app/views/users/index.html.erb

```diff
@@ -1,12 +1,82 @@
 <section class="users-section">
   <ul class="users-list">
     <% @users.each do |user| %>
+
+    <%if user.id != current_user.id %>
       <li>
-        Name: <%= user.name %>
+        Name: <%= user.name %> 
         <span class="profile-link">
+
           <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+         
+          <span class=" button is-link">  <%= friendship(user.id) %></span>
         </span>
       </li>
+      <% end %>
+    <% end %>
+  </ul>
+
+
+</section><h1>My friends</h1>
+
+<section class="users-section">
+  <ul class="users-list">
+    <% if !@friends.nil? %>
+    <% @friends.each do |user| %>
+      <li>
+        <p></p>
+        Name: <%= user.name %> 
+        <span class="profile-link">
+
+          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          <span class= "button is-success"> friends</span>
+          <span class=" button is-danger"><%= destroy_friendship(user.id) %> </span>
+        </span>
+      </li>
+    <% end %>
     <% end %>
```

Comment:
- [ ]  Put this if statement in a helper method that you call here and put the contents of the if statement in a partial that you render within the helper method to avoid putting the logic straight in the views.